winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,176 @@
 ---
 layout: post
 title: Customization in WinUI AvatarView control | Syncfusion
-description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+description: Learn all about the various customization features available in the WinUI AvatarView control, including border, background, gradient, font, and more.
```

Comment:
Description: Learn about the various customization features in the WinUI AvatarView control, including border, background, gradient, font, and more.